core/src/main/java/com/datastrato/gravitino/GravitinoEnv.java

```diff
@@ -104,10 +100,11 @@ public Config config() {
   }
 
   /**
-   * Get the EntitySerDe associated with the Gravitino environment.
-   *
-   * @return The EntitySerDe instance.
+   * @deprecated Put the {@link EntitySerDe} initialization logic of the {@link EntityStore} into
+   *     the {@link EntityStore#initialize} method. If you need to get the {@link EntitySerDe} of
+   *     the {@link EntityStore}, you can add a method in {@link EntityStore}.
    */
+  @Deprecated
   public EntitySerDe entitySerDe() {
     return entitySerDe;
   }
```

Comment:
Is this still valid, seem it will return null? It is not what `@Deprecated` means. `Deprecation` means that the method can still be used and the semantics are the same, but recommended to use another API. But here, the semantics are changed, this API cannot be used anymore, am I right?

Reply:
I think so, it will return `Null`. I checked that this method is not called elsewhere, maybe it can be deleted directly.

Reply:
You can delete it.

Reply:
Yes, I think you can delete it.

Reply:
Done